### ISSUE TITLE
Synchronize with openBIS once a night

### ISSUE THREAD

**Issue by julianspaeth** [qbicsoftware/project-manager#73]
The project overview database should be updated once a night.

**Comment by julianspaeth** [comment]
@FriederikeHanssen i think you want to implement or already implemented something similar?

**Comment by FriederikeHanssen** [comment]
Yes, something similar. For the statistics portlet we divided the querying and visualization into two tools. One of them is responsible for querying opens etc periodically and writing the results to a yaml file. The other takes any existing yaml file and visualises the respective data. Thus, if the queries fail for some reason, an earlier yaml can be used for visualisation as well. The periodic querying will not be done from within java. Rather the java code will be executed periodically. 

See here :
query tool
https://github.com/qbicsoftware/statistics-data-retrieval
and here: 
visualization portlet: 
https://github.com/qbicsoftware/qbic-statistics
